sw/device/lib/dif/BUILD

```diff
@@ -303,9 +302,11 @@ cc_library(
         "dif_hmac.c",
     ],
     hdrs = [
+        "autogen/dif_hmac_autogen.h",
```

Comment:
I believe the autogen headers are considered private headers and should go in the `srcs` section.